test/logs/ARM_MUSCA_S1/CRYPTO.log

```diff
@@ -0,0 +1,6 @@
+Crypto Suite Report
+TOTAL TESTS     : [0-9]+
+TOTAL PASSED    : [0-9]+
+TOTAL SIM ERROR : 0
+TOTAL FAILED    : 27
+TOTAL SKIPPED   : 1
```

Comment:
[Musca_S1_GNUARM_CRYPTO.log](https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/files/5224056/GNUARM_CRYPTO.log) refers that there are no skipped tests, is there a reason one test skip is to be considered here?

Reply:
This must have been a typo, it is changed.